### PR TITLE
[EOS-22988] m0kv -help throwing ERROR in normal run

### PR DESCRIPTION
**Pull request by VinothVijayakumar-dev**
- exited the flow for the help options
	  ( utilites updated - m0kv , m0add2dump )

Signed-off-by: Vinoth <Vinoth.V@seagate.com>

### CODE REVIEW

File: motr/m0kv/cmd_main.c [hunk 1]
```diff
@@ -136,12 +136,18 @@ static int opts_get(struct params *par, int *argc, char ***argv)
...
+		exit(EXIT_FAILURE);;
+	}
+
 	/* All mandatory params must be defined. */
 	if (rc == 0 &&
 	    (par->cp_local_addr == NULL || par->cp_ha_addr == NULL ||
 	     par->cp_prof == NULL || par->cp_proc_fid == NULL)) {
 		usage();
-		rc = M0_ERR(-EINVAL);
+		exit(EXIT_FAILURE);;
```
Comment (ankit-seagate) [inline motr/m0kv/cmd_main.c:150]: If any of the parameter is NULL, then it should post the error, right?

**Review by ankit-seagate — APPROVED** [review]
Looks fine

**Review by atulsdeshmukh2312 — APPROVED** [review]

**Review by ankit-seagate — APPROVED** [review]
Approved

**Review by ankit-seagate — APPROVED** [review]
+1

**Pull request merged**